src/Faithlife.Tracing.Http/HttpTracingUtility.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+
+namespace Faithlife.Tracing.Http
+{
+	public static class HttpTracingUtility
+	{
+		public static Func<HttpWebRequest, IDisposable> CreateStartTrace(ITraceProvider traceProvider)
+		{
+			return r =>
+			{
+				var currentTrace = traceProvider?.CurrentTrace;
+				if (currentTrace == null)
+				{
+					r.Headers[HttpHeaderNames.B3Sampled] = "0";
```

Comment:
Why is this needed?

Reply:
If there's no trace, this service has (possibly implicitly) made a reject sampling decision, so it gets propagated: https://github.com/openzipkin/b3-propagation#why-propagate-a-reject-sampling-decision